WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -399,6 +410,7 @@ public void onClick(View view) {
             }
         });
 
+        if (getEditPostRepository() != null) hideSpecificViews(getEditPostRepository().isPage());
```

Comment:
Sure, will do that change!

I usually try to follow the general Kotlin [style guides for braces](https://developer.android.com/kotlin/style-guide#braces) and end up having those single line ifs some times, lol, but I am happy to change to the pattern used in the file.

Reply:
Yeah, I would also normally not ask for such changes since I'm also usually in favor of the Kotlin style guides 😅; but for guidelines priority to I rank higher respecting the patterns specific to a context (in this situation a Java file that already has enough code with its own rules).
Thank you 🙇

Reply:
Done!